debian/ibverbs-providers.maintscript

```diff
@@ -1,2 +1,3 @@
 rm_conffile /etc/libibverbs.d/cxgb3.driver 27.0-2~
+rm_conffile /etc/libibverbs.d/i40iw.driver
```

Comment:
Can you add `39` as `prior-version`? See https://manpages.debian.org/jessie/dpkg/dpkg-maintscript-helper.1.en.html for the explanation.

Reply:
Added